.github/workflows/rust.yml

```diff
@@ -39,3 +39,5 @@ jobs:
         continue-on-error: true
       - name: Format
         run: cargo fmt --all -- --check
+      - name: Test (integration test feature)
```

Comment:
This is awkward because it's different from how we run tests for the default features case, but I don't see the point of generating another coverage report.

Reply:
at the very least we could use `cargo nextest`; it's faster and has nicer output